src/build-extensions-container.sh

```diff
@@ -33,7 +34,9 @@ img=localhost/extensions-container
 (set -x; podman build --from oci-archive:"$ostree_ociarchive" --network=host \
     --build-arg COSA=true --build-arg VARIANT="${variant}" --label version="$buildid" \
     --volume /etc/pki/ca-trust:/etc/pki/ca-trust:ro \
+    --volume "${ctx_dir}/package_extensions:/extensions" \
     -t "${img}" -f extensions/Dockerfile "${ctx_dir}")
+mv "${ctx_dir}/package_extensions/extensions.json" ./tmp
```

Comment:
instead of this musical chairs would it work to just create the file (i.e. make it empty) and then bind mount it directly into the build?

```
> ./tmp/extensions.json
podman build...
   --volume ./tmp/extensions.json:/tmp/extensions.json
```

Then you don't need to `mv` anything at the end.